config/ldap_auth.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+    'identifiers' => [
+        'ldap' => [
+            'locate_users_by' => 'mail',
+        ],
+        'database' => [
+            'guid_column' => 'email',
```

Comment:
Heads-up: this is a bit of a hack here. It enables us to *not* have to create a column in our user table to store the LDAP objectid identifier. For posterity: the effect it has is when locating the LDAP user in the CDash database, it will query the database with a `WHERE` clause similar to:
```sql
SELECT * FROM `user`
 WHERE `user`.`email` = '2e54e696-1333-413f-9211-21e212cae55b'
    OR `user`.`email` = 'ldap.user@company.tld'
```